<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanConfigJ1939AdditionalsDialog</class>
 <widget class="QDialog" name="CanConfigJ1939AdditionalsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>J1939 additional configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="VariableDlcCheckBox">
       <property name="text">
        <string>Variable DLC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="VariableDlcLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Init value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="VariableDlclInitValueLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Destination Address</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="DestAddrFixedRadioButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Fixed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TextValueInput" name="DestAddrFixedLineEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="DestAddrVariableRadioButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DestAddrVariableLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Init value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TextValueInput" name="DestAddrVariableInitValueLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextValueInput</class>
   <extends>QLineEdit</extends>
   <header>TextValueInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanConfigJ1939AdditionalsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanConfigJ1939AdditionalsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
